app/validation/modify_grow.py

```diff
@@ -0,0 +1,41 @@
+from typing import Any, Dict, List, Tuple, Optional
+from collections import Counter
+
+from app_config import AppConfig
+
+from app.models.grow import Grow
+from app.models.grow_phase import GrowPhase
+from app.models.recipe import Recipe
+from app.models.recipe_phase import RecipePhase
+
+
+def validate_modify_grow(app_config: AppConfig, message) -> Tuple[bool, str]:
```

Comment:
Add type for all args

Reply:
Also add a function level comment with a comment for all of the args

Reply:
Have you added this validation to the `modify_grow` event? @egba4444